openmc/mgxs/__init__.py

```diff
@@ -42,3 +44,409 @@
+    1.3804e-1, 1.4454e-1, 1.5136e-1, 1.5849e-1, 1.6596e-1, 1.7378e-1,
+    1.8197e-1, 1.9055e-1, 1.9953e-1, 2.0893e-1, 2.1878e-1, 2.2909e-1,
+    2.3988e-1, 2.5119e-1, 2.6303e-1, 2.7542e-1, 2.8840e-1, 3.0200e-1,
+    3.1623e-1, 3.3113e-1, 3.4674e-1, 3.6308e-1, 3.8019e-1, 3.9811e-1,
+    4.1687e-1, 4.3652e-1, 4.5709e-1, 4.7863e-1, 5.0119e-1, 5.2481e-1,
+    5.4954e-1, 5.7544e-1, 6.0256e-1, 6.3096e-1, 6.6069e-1, 6.9183e-1,
+    7.2444e-1, 7.5858e-1, 7.9433e-1, 8.3176e-1, 8.7096e-1, 9.1201e-1,
+    9.5499e-1, 1.0000, 1.0471, 1.0965, 1.1482, 1.2023,
+    1.2589, 1.3183, 1.3804, 1.4454, 1.5136, 1.5849,
+    1.6596, 1.7378, 1.8197, 1.9055, 1.9953, 2.0893,
+    2.1878, 2.2909, 2.3988, 2.5119, 2.6303, 2.7542,
+    2.8840, 3.0200, 3.1623, 3.3113, 3.4674, 3.6308,
+    3.8019, 3.9811, 4.1687, 4.3652, 4.5709, 4.7863,
+    5.0000, 5.2000, 5.4000, 5.6000, 5.8000, 6.0000,
+    6.2000, 6.4000, 6.6000, 6.8000, 7.0000, 7.2000,
+    7.4000, 7.6000, 7.8000, 8.0000, 8.2000, 8.4000,
+    8.6000, 8.8000, 9.0000, 9.2000, 9.4000, 9.6000,
+    9.8000, 1.0000e1, 1.0200e1, 1.0400e1, 1.0600e1, 1.0800e1,
+    1.1000e1, 1.1200e1, 1.1400e1, 1.1600e1, 1.1800e1, 1.2000e1,
+    1.2200e1, 1.2400e1, 1.2600e1, 1.2800e1, 1.3000e1, 1.3200e1,
+    1.3400e1, 1.3600e1, 1.3800e1, 1.4000e1, 1.4200e1, 1.4400e1,
+    1.4600e1, 1.4800e1, 1.5000e1, 1.5200e1, 1.5400e1, 1.5600e1,
+    1.5800e1, 1.6000e1, 1.6200e1, 1.6400e1, 1.6600e1, 1.6800e1,
+    1.7000e1, 1.7200e1, 1.7400e1, 1.7600e1, 1.7800e1, 1.8000e1,
+    1.8200e1, 1.8400e1, 1.8600e1, 1.8800e1, 1.9000e1, 1.9200e1,
+    1.9400e1, 1.9600e1, 1.9800e1, 2.0000e1, 2.0200e1, 2.0400e1,
+    2.0600e1, 2.0800e1, 2.1000e1, 2.1200e1, 2.1400e1, 2.1600e1,
+    2.1800e1, 2.2000e1, 2.2200e1, 2.2400e1, 2.2600e1, 2.2800e1,
+    2.3000e1, 2.3200e1, 2.3400e1, 2.3600e1, 2.3800e1, 2.4000e1,
+    2.4200e1, 2.4400e1, 2.4600e1, 2.4800e1, 2.5000e1, 2.5200e1,
+    2.5400e1, 2.5600e1, 2.5800e1, 2.6000e1, 2.6200e1, 2.6400e1,
+    2.6600e1, 2.6800e1, 2.7000e1, 2.7200e1, 2.7400e1, 2.7600e1,
+    2.7800e1, 2.8000e1, 2.8200e1, 2.8400e1, 2.8600e1, 2.8800e1,
+    2.9000e1, 2.9200e1, 2.9400e1, 2.9600e1, 2.9800e1, 3.0000e1,
+    3.0200e1, 3.1623e1, 3.3113e1, 3.4674e1, 3.6308e1, 3.8019e1,
+    3.9811e1, 4.1687e1, 4.3652e1, 4.5709e1, 4.7863e1, 5.0119e1,
+    5.2481e1, 5.4954e1, 5.7544e1, 6.0256e1, 6.3096e1, 6.6069e1,
+    6.9183e1, 7.2444e1, 7.5858e1, 7.9433e1, 8.3176e1, 8.7096e1,
+    9.1201e1, 9.5499e1, 1.0000e2, 1.0471e2, 1.0965e2, 1.1482e2,
+    1.2023e2, 1.2589e2, 1.3183e2, 1.3804e2, 1.4454e2, 1.5136e2,
+    1.5849e2, 1.6596e2, 1.7378e2, 1.8197e2, 1.9055e2, 1.9953e2,
+    2.0893e2, 2.1878e2, 2.2909e2, 2.3988e2, 2.5119e2, 2.6303e2,
+    2.7542e2, 2.8840e2, 3.0200e2, 3.1623e2, 3.3113e2, 3.4674e2,
+    3.6308e2, 3.8019e2, 3.9811e2, 4.1687e2, 4.3652e2, 4.5709e2,
+    4.7863e2, 5.0119e2, 5.2481e2, 5.4954e2, 5.7544e2, 6.0256e2,
+    6.3096e2, 6.6069e2, 6.9183e2, 7.2444e2, 7.5858e2, 7.9433e2,
+    8.3176e2, 8.7096e2, 9.1201e2, 9.5499e2, 1.e3])
+    
```

Comment:
Is this GitHub telling me there is or is not a blank line at the end of the file?

Either way, can you double check and make sure there is one?

Reply:
checked